public/components/wz-search-bar/components/wz-search-badges/wz-search-badges.tsx

```diff
@@ -9,31 +9,39 @@
  *
  * Find more information about this on the LICENSE file.
  */
-import React, { } from 'react';
+import React from 'react';
 import { EuiBadge } from '@elastic/eui';
-import { GroupingComponents} from '../../../common/util';
+import { GroupingComponents } from '../../../common/util';
 import './wz-search-badges.scss';
+export const WZ_SEARCH_BADGE_NAME = 'wz-search-badge';
 
 const buttonLabel = (count) => `+${count} filters`;
 
 export function WzSearchBadges({ filters, onFiltersChange }) {
-  const removeFilter = (key) => { const newFilters = [...filters]; newFilters.splice(key, 1); onFiltersChange(newFilters) };
+  const removeFilter = (key) => {
+    const newFilters = [...filters];
+    newFilters.splice(key, 1);
+    onFiltersChange(newFilters);
+  };
   const badges = filters.map((filter, key) => badge({ filter, key, removeFilter }));
-  const gruopingBadges = GroupingComponents({children: badges, buttonLabel})
+  const gruopingBadges = GroupingComponents({ children: badges, buttonLabel });
   return gruopingBadges;
 }
 
 function badge({ filter, key, removeFilter }) {
   return (
-    <EuiBadge key={key}
-      className='wz-search-badge'
-      color='hollow'
-      iconType='cross'
-      iconSide='right'
-      onFocus={e => e.stopPropagation()}
+    <EuiBadge
+      key={key}
+      data-testid={`${WZ_SEARCH_BADGE_NAME}-${key}`}
```

Comment:
**question**: What this attribute is used for? If it is related to testing, we use `data-test-subj`.

Reply:
It is used only for testing purposes, using `testing-library/react` and that property I can get the element reference to test it. Only works with data-testid